extensions/wikia/UserLogin/FacebookSignupController.class.php

```diff
@@ -162,6 +159,8 @@ public function modal() {
 			'returnToUrl' => $returnToUrl,
 			'queryString' => $returnToParams,
 			'loginToken' => UserLoginHelper::getSignupToken(),
+			'usernameKey' => UserLoginForm::SIGNUP_USERNAME_KEY,
+			'passwordKey' => UserLoginForm::SIGNUP_PASSWORD_KEY,
```

Comment:
It is a good move to get rid of the consts from this module and unify them. 
And what do you think about not importing these from 'sibling' module which has different responsibility?. Ideally would be that both modules (FacebookSignupController and UserLoginForm) inherit these consts from one place in UserLogin extension.